config/settings.yml

```diff
@@ -109,6 +109,17 @@
     :additional_regions: {}
   :ems_azure:
     :disabled_regions: []
+    :api_versions:
+      :availability_set: 2017-03-30
```

Comment:
With the old settings structure there was a mechanism to "ensure" new settings.  Not sure if that's needed with the new settings anymore.